plugins/initLaunchDarkly.ts

```diff
@@ -0,0 +1,22 @@
+import { initLdClient } from '@/utils'
+import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
+
+/**
+ * This plugin function is called before instantiating the root Vue.js application.
+ * It initializes LaunchDarkly.
+ */
+ export default async function (context) {
+   // save id to window object for init function
+  window['ldClientId'] = context.$config.ldClientId
+
+  if (window['ldClientId']) {
+    console.info('Initializing LaunchDarkly...')
+
+    // initialize LD using local library
+    const ldClient = await initLdClient()
+
+    // also save flags in session for common components (eg, SbcHeader)
+    const allFlags = JSON.stringify(ldClient.allFlags())
+    sessionStorage.setItem(SessionStorageKeys.LaunchDarklyFlags, allFlags)
+  }
+}
```

Comment:
This kind of sucks but it's the cleaner solution atm...

The problem is that SbcHeader uses a different service for getting the LD feature flags.

So I use the "good" library to init LD in this project, and the store the flags in session storage for SbcHeader to use.

Some day, it would be nice if SbcHeader used the "good" library, which fetches the latest flag value from the LD client object (which is kept synced with the LD server in the background).